assets/src/styles/blocks/Spreadsheet/SpreadsheetStyle.scss

```diff
@@ -38,16 +38,9 @@ table.spreadsheet-table {
     overflow: visible;
   }
 
-  // Grey is the default color.
-  th --block-spreadsheet-grey-header-- {
-    background: $table-header-grey;
-  }
-
   tr {
     td {
-      --block-spreadsheet-- {
-        color: $table-font-color;
-      }
+      color: var(--block-spreadsheet--color, $table-font-color);
```

Comment:
Maybe here we could just use `var(--body--color)` and it will be directly updated, same thing for the Table block [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/layout/_tables.scss#L45) (I wonder why we have the styles for that block split between 2 files?!). That would also allow us to get rid of this [variable](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_colors.scss#L57). What do you think?

Reply:
If i am not mistaken, `var(--body--color)` exist only when the new identity styles is enabled right? So we may want to do something like `color: var(--body--color, $table-font-color)`, so we still maintain the old styles when we switch back.